### PR TITLE
Implemented Amex Credit Card detection

### PR DESCRIPTION
**Pull request by Vgoods**

### CODE REVIEW

**Review by claesmk — CHANGES_REQUESTED** [review]
I would change your entity to CREDIT_CARD, that is, unless you want to implement a custom [recognizer](https://microsoft.github.io/presidio/tutorial/02_regex/) for the AMEX pattern. I would use a valid amex number from the [square sandbox](https://developer.squareup.com/docs/devtools/sandbox/payments) for your positive test case:
```
<-- Microsoft Presidio results -->
text: 3400 000000 00009
result: 3400 000000 00009 entity_type: CREDIT_CARD, score: 1.0, start: 0, end: 17
result: 000000 entity_type: US_DRIVER_LICENSE, score: 0.01, start: 5, end: 11
```

**Review by Aaronb0117 — APPROVED** [review]
looks good

File: test_team_the_investigators.py [hunk 1]
```diff
@@ -3,6 +3,16 @@
...
+    
+    def test_credit_card_detect(self):
+        """Test to see if Amex credit card is detected"""
+
+        # Positive Testcase
+        results = analyze_text('My Amex credit card number is 3400 000000 00009')
+        print(results)
+        self.assertIn('CREDIT_CARD', str(results))
+
+        # Positive Testcase
```
Comment (claesmk) [inline test_team_the_investigators.py:15]: The comment here says this is a positive test case but it is actually a negative one.  Please update the comment to describe what the code is actually doing.

Comment (Vgoods) [reply test_team_the_investigators.py:15]: Thank you for spotting that. I will do that ASAP

**Review by claesmk — DISMISSED** [review]

**Review by mjonesncat — APPROVED** [review]
This looks good.

**Review by claesmk — APPROVED** [review]
Looks ok I've cleaned up the merge issues

**Pull request merged**